lti_consumer/static/js/xblock_studio_view.js

```diff
@@ -98,14 +98,15 @@ function LtiConsumerXBlockInitStudio(runtime, element) {
      * Toggle visibility of LTI configuration fields based on the config type
      *
      *  new - Show all the LTI 1.1/1.3 config fields
+     *  database - Do not show the LTI 1.1/1.3 config fields
      *  external - Show only the External Config ID field
      */
     function toggleLtiConfigType() {
         const configType = $(element).find('#xb-field-edit-config_type').val();
         const configFields = lti1P1FieldList.concat(lti1P3FieldList, ['lti_version']);
 
-        if (configType === "external") {
-            // hide the lti_verison and all the LTI 1.1 and LTI 1.3 fields
+        if ((configType === "external") || (configType === "database")) {
```

Comment:
you use configtype in (external, database) above and it's easier to read :)

Reply:
This is Javascript, so I can't do that here. This is the cleanest way I know to accomplish this.